BSP/Simulator/Src/BSP_CAN.c

```diff
@@ -12,22 +12,36 @@
 #include <unistd.h>
 
 static bool CAN_Initialized = false;
+static bool FaultState = false;
```

Comment:
what is this variable for?

Reply:
fixed

Reply:
How was this fixed? I don't see the variable used anywhere except for it being set in `BSP_CAN_Init`?